install/source/index.markdown

```diff
@@ -100,61 +100,45 @@ Note, every command to ``catkin config --blacklist`` will override the previous
 
 ## Optional: Install a Compiler Cache
 
-Even if you decide to not build *all* of MoveIt, building it from source can still take quite some time.
-Because of this, it can be beneficial to use a tool called a *compiler cache* to avoid recompiling files that have not been changed between builds but which with a regular build (without such a cache) would still be processed by the compiler.
-This situation is especially common when switching between different branches while developing and when `clean`ing a Catkin workspace and then rebuilding it.
+Building *all* or even just *some* of MoveIt from source can take up to an hour.
+A compiler cache can reduce this time to a few minutes by preventing unchanged files from being recompiled.
+The most prominent compiler cache is [ccache](https://ccache.dev) suitable for `gcc` and other similar compilers.
 
-[ccache](https://ccache.dev) provides such a compiler cache and using it can reduce the time needed to rebuild MoveIt (and other packages) from many minutes to mere seconds.
-Note: `ccache` is only compatible with GCC and only caches (Obj)C/(Obj)C++ output, but as MoveIt is largely a C++ application it can benefit greatly.
-
-**Note**: as `ccache` uses the filesystem to cache objects, be sure to have sufficient free space available.
-The default maximum cache size on Debian and Ubuntu is up to 5 GB for a full cache (if there are no files in the cache, it will not take up any space).
-Configure a different maximum with the `--max-size` option.
-
-Refer to the [ccache website](https://ccache.dev) for more information on when the cache is used and how to change its configuration.
-
-### Installation (Debian/Ubuntu)
+### Installation
 
 On a Debian or Ubuntu system, installing `ccache` is simple:
 
     sudo apt-get install ccache
 
-### Installation (other systems)
-
-On other OS, consult the package manager or software store and search for `ccache`.
+For other OS, consult the package manager or software store and search for `ccache`.
 Refer to the [ccache website](https://ccache.dev) for more information on downloading and installation.
```

Comment:
the link cache.dev is listed 3x here. I think this sentence referencing it can just be removed.
```suggestion
```

Reply:
Removed